tribunestyles.html

```diff
@@ -0,0 +1,218 @@
+---
+layout: default
+title: TexasTribune
+---
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/tribune.min.css" >
+    <script type="text/javascript" src="//use.typekit.net/hxp4tcr.js"></script>
```

Comment:
so this page reflects production closer, should use `//use.typekit.com/brb4jmm.js` instead (may need to adjust our settings for that account).